webpack.config.babel.js

```diff
@@ -100,20 +100,27 @@ module.exports = {
 		}),
 		new HtmlWebpackPlugin({
 			template: './index.ejs',
-			minify: { collapseWhitespace: true }
+			minify: { collapseWhitespace: true },
+			filename: 'index.html'
+		}),
+		new HtmlWebpackPlugin({
+			template: './200.ejs',
+			minify: { collapseWhitespace: true },
+			filename: '200.html'
```

Comment:
any chance it'd be possible to use the existing `CopyWebpackPlugin` to just clone `index.html` as `200.html`? (not sure)